cmd/ovsdb-mon/tables.go

```diff
@@ -63,16 +63,16 @@ func NewStructPrinter(writer io.Writer, stype reflect.Type, fieldSel ...string)
 		for _, sel := range fieldSel {
 			found := false
 			for i := 0; i < stype.NumField(); i++ {
-				if stype.Field(i).Name == sel {
+				if stype.Field(i).Name == sel || strings.ToLower(stype.Field(i).Name) == sel {
```

Comment:
NIT: how about using EqualFold ?
```
if strings.EqualFold(stype.Field(i).Name, sel) {
```
https://golangbyexample.com/golang-case-insensitive-string-comparison/

Reply:
Yes, we can use EqualFold. However, the autocomplete list will only have the ToLower version of the column name so unless the user goes wild with casing without the assistance of the autocomplete, it wouldn't actually make much difference.